python/pip_install/pip_repository.bzl

```
@@ -36,6 +36,20 @@ def _parse_optional_attrs(rctx, args):
     Returns: Augmented args list.
     """
 
+    # Determine whether or not to pass the pip `--isloated` flag to the pip invocation
+    use_isolated = rctx.attr.isolated
+
+    # The environment variable will take precedence over the attribute
+    isolated_env = rctx.os.environ.get("RULES_PYTHON_PIP_ISOLATED", None)
+    if isolated_env != None:
+        if isolated_env.lower() in ["", "0", "false"]:
```

The empty string doesn't make sense as a falsey value to me. Can we keep this to set values?
```suggestion
        if isolated_env.lower() in ("0", "false"):
```

---

Yeah, sounds good to me 😃